analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -135,7 +136,8 @@ class Gradle : PackageManager() {
             )
 
             // Gradle < 4.8 does not support lock files, so hard-code "allowDynamicVersions" to "true".
-            return ProjectAnalyzerResult(true, project,
+            val config = AnalyzerConfiguration(false, true)
```

Comment:
This also hard-codes `ignoreToolVersions` in the result. I'm not sure if we need the value of that flag in the analyzer result, but if we do, I think it should be correct for all package managers.

Reply:
Done.